Runtime/Components/ConeRayAnglesEstimator/GuidedDataCollector.cs

```diff
@@ -50,17 +86,35 @@ public void EndCalibrationForSegment(HPUIInteractorConeRayAngleSegment segment)
 
         /// <summary>
         /// This will create a <see cref="ConeRayComputationDataRecord"/> for the
-        /// current <see cref="TargetSegment"/>.
+        /// current <see cref="TargetSegment"/>. Also pauses the data collector
+        /// to adjust target segment for the next data record.
         /// </summary>
         public void EndDataCollectionForTargetSegment()
         {
             EndCalibrationForSegment(TargetSegment);
             PauseDataCollection = true;
         }
 
+        /// <summary>
+        /// Resumes data collection. To be used after 
+        /// `EndDataCollectionForTargetSegment`
+        /// </summary>
         public void StartDataCollectionForNextTargetSegment()
         {
             PauseDataCollection = false;
         }
+
+        /// <summary>
+        /// Set's the target segment to be calibrated.
+        /// See <see cref="GuidedDataCollectorEditor.StepThroughOrderOfCalibrationPhalanges"/> 
+        /// for an example on using this during calibration procedures.
+        /// </summary>
+        /// <param name="parameterName">Parameter description.</param>
+        /// <returns>Type and description of the returned object.</returns>
+        /// <example>Write me later.</example>
+        public void StepToTargetPhalange(HPUIInteractorConeRayAngleSegment targetSegment)
+        {
+            this.targetSegment = targetSegment;
```

Comment:
The setter logic of the prop doesn't get run here? should this method be removed in favour of the prop?